generators/guestcontainer/index.js

```diff
@@ -179,6 +179,26 @@ module.exports = generators.Base.extend({
                         appTypeName: appTypeName
                         }
                 );
+
```

Comment:
Once we update the original install and uninstall script, there would not be any change required here in this file, I guess.